client/dashboard/app/queries/me-transferred-purchases.ts

```diff
@@ -0,0 +1,12 @@
+import { queryOptions } from '@tanstack/react-query';
+import { fetchTransferredPurchases } from '../../data/me-transferred-purchases';
+
+const queryKey = 'transferred-purchases';
```

Comment:
Team review: Consider inlining constant strings for query keys for better type inference.